services/workflows-service/prisma/migrations/20240311133449_add_label_to_alert_definitions/migration.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "AlertDefinition" ADD COLUMN "label" TEXT NOT NULL DEFAULT substr(md5(random()::text), 0, 10);
```

Comment:
Added a default to not break local environments that already have alert definitions